### PR TITLE
PP-43: corrections to Refactor Register Relay

### PR DESCRIPTION
**Pull request by mortelli**
## What

- udpate code based on `rif-relay-contracts` and `rif-relay-common` changes (see _Refs_ section)

## Why

- code inspection has shown some errors in the current implementation

## Refs

- [PP-43](https://rsklabs.atlassian.net/browse/PP-43)
- [Refactor Register Relay: changes to be implemented](https://hackmd.io/XS3gcRwjQxe9qetWQ4o8vw?view)
- `rif-relay-contracts`: [PP-43: corrections to Refactor Register Relay](https://github.com/anarancio/rif-relay-contracts/pull/9)
- `rif-relay-common`: [PP-43: corrections to Refactor Register Relay](https://github.com/infuy/rif-relay-common/pull/8)

### CODE REVIEW

File: src/KnownRelaysManager.ts [hunk 1]
```diff
@@ -73,14 +73,14 @@ export class KnownRelaysManager {
...
-        // As 'topics' are used as 'filter', having an empty set results in querying all register events.
+    ): Promise<RelayManagerData[]> {
         if (relayManagers.size === 0) {
             return [];
         }
-        const activeRelays = await this.contractInteractor.getActiveRelays(
+        const activeRelays = await this.contractInteractor.getActiveRelayInfo(
             relayManagers
         );
+        // As 'topics' are used as 'filter', having an empty set results in querying all register events.
```
Comment (mortelli) [inline src/KnownRelaysManager.ts:83]: comment moved as it appeared unrelated to line `77`

**Review by antomor — APPROVED** [review]
LVGTM!
